smol/cofe/space/cluster.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, site_spaces, frac_coords, lattice):
         self._centroid = centroid - shift
         self._frac_coords = frac_coords - shift
         self._sites = tuple(
-            Site(site_space, coords)
+            Site(Composition({sp: amt for sp, amt in site_space.items()}), coords)
```

Comment:
Hi, just a little comment here: changing site_space into Composition could be a quick work-around, but it could be better if we simply use site_space.composition instead. Also, have we found the root cause of this problem? Is it due to an incorrect parsing of Vacancy?

Reply:
That is a good point. The root cause is that get_el_sp() (in pymatgen.core.periodic_table) is no longer able to take a SiteSpace instance as an input. And get_el_sp() is used to instantiate a Site when the input species is not a Species or Composition instance.
I've tried a prim without Vacancy which runs into the same error, so Vacancy is not the issue. 
This all arises likely from some change in pymatgen, but haven't been able to source exactly what has changed.

Reply:
Thank you for finding out!